src/components/NewConversationDialog/NewConversationDialog.vue

```diff
@@ -293,9 +294,11 @@ export default {
 			this.page = 2
 
 			try {
+				const emailParticipants = this.selectedParticipants.filter(participant => participant?.source === ATTENDEE.ACTOR_TYPE.EMAILS)
+
 				this.newConversation.token = await this.$store.dispatch('createGroupConversation', {
 					conversationName: this.conversationName,
-					isPublic: this.isPublic,
+					isPublic: this.isPublic || emailParticipants.length,
```

Comment:
This should not be done. Let's delay the publishing until the API did it?

Reply:
Also fine, would be future-proof in case we allow emails in private rooms